src/AltinnCore/UnitTest/Helpers/IGiteaMockHelper.cs

```diff
@@ -62,14 +62,14 @@ public static void GetCurrentUser_ReturnsOne(Mock<IGitea> moqGiteaWrapper)
             moqGiteaWrapper.Setup(gitea => gitea.GetCurrentUser()).ReturnsAsync(user);
         }
 
-        /// <summary>
-        /// Add a Mock organization to the mock
-        /// </summary>
-        /// <param name="moqGiteaWrapper">The Gitea mock</param>
-        public static void AddOneOrg(Mock<IGitea> moqGiteaWrapper)
-        {
-            Organization organization = new Organization() { Username = "MockOrg" };
-            moqGiteaWrapper.Setup(gitea => gitea.GetOrganization(It.IsAny<string>())).ReturnsAsync(organization);
-        }
+        /////// <summary>
```

Comment:
Is there a specific reason we aren't deleting these code lines but rather commenting them out?

Reply:
Good catch, deleting!